src/main/java/models/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package models;
+
+import core.basesyntax.ColorSupplier;
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
```

Comment:
method should not be static
```suggestion
    public Ball getRandomBall() {
```

Reply:
changed